frontend/src/AlertsManagement/ManagementRoutes.test.js

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import { configure, mount } from "enzyme";
+import Adapter from 'enzyme-adapter-react-16';
+import AlertsContent from './AlertsContent';
+import AlertInfo from './AlertInfo';
+import ManagementRoutes from "./ManagementRoutes";
+import { MemoryRouter } from "react-router-dom";
+
+configure({ adapter: new Adapter() });
+
+describe("navigating in AlertsManagement", () => {
+  it("should display AlertsContent component on route", () => {
+    const wrapper = mount(
+      <MemoryRouter initialEntries={[ '/alerts' ]} initialIndex={0}>
```

Comment:
ooc, what does `MemoryRouter` do?

is it some sort of mock router used for test?

Reply:
yup, `MemoryRouter` is a mock Router you can use for tests because you need to have a `Router` in order to mount the Routes. you can mock the initial route(s) with `initialEntries`

Reply:
thanks for the explanation!